src/BenchmarksServer/Startup.cs

```diff
@@ -211,16 +212,57 @@ private static string CloneAndRestore(string path, ServerJob job)
 
             Debug.Assert(benchmarksDir != null);
 
-            // TODO: add project references to checked out repos
-            // https://github.com/aspnet/benchmarks/issues/185
+            AddSourceDependencies(path, benchmarksDir, dirs);
 
-            // Restore in each dir
-            foreach (var dir in dirs)
+            ProcessUtil.Run("dotnet", "restore /p:VersionSuffix=zzzzz-99999", workingDirectory: Path.Combine(path, benchmarksDir));
```

Comment:
The version suffix needs to be higher than whatever package versions we have in our feed. Hence this thing. I tried coming up with something whimsical but decided to go with this boring suffix 😛